packages/client/hmi-client/src/assets/css/theme/_asset.scss

```diff
@@ -0,0 +1,95 @@
+.asset {
+    display: flex;
+    flex-direction: column;
+
+    div,
+    span {
+        overflow-wrap: break-word;
+    }
+
+    header {
+        margin: 0rem 1rem;
+        color: var(--text-color-subdued);
+        display: flex;
+        flex-direction: column;
+        gap: 0.5rem;
+
+        .authors,
+        .journal,
+        .framework,
+        .simulation {
+            color: var(--primary-color-dark);
+        }
+
+        h4 {
+            color: var(--text-color-primary);
+        }
+
+        .header-and-buttons {
+            display: flex;
+            justify-content: space-between;
+
+            button {
+                margin: 0 0.25rem
+            }
+        }
+
+        button {
+            width: fit-content;
+        }
+    }
+
+    .p-accordion {
+        margin: 1rem 0.5rem;
+        margin-top: 1rem;
+
+        ul {
+            list-style: none;
+
+            li {
+                margin-bottom: 0.5rem;
+            }
+        }
+
+        p {
+            margin: 0.5rem 0;
+        }
+
+        strong {
+            font-weight: $fontWeightSemiBold;
```

Comment:
I know you use SCSS to leverage nesting, and that's fine here, but we should really use only CSS custom properties.
That will make our life easier if in the future we can get rid of SCSS completely.